src/api/hooks/useGetBlock.ts

```diff
@@ -4,23 +4,23 @@ import {getBlockByHeight, getBlockByVersion} from "../../api";
 import {ResponseError} from "../../api/client";
 import {useGlobalState} from "../../GlobalState";
 
-export function useGetBlockByHeight(height: number) {
+export function useGetBlockByHeight(height: number, withTransactions = true) {
```

Comment:
I recommend using a keyword argument instead of a positional argument for this. That will make it clearer at the callsite what true/false means.